.travis.yml

```diff
@@ -9,6 +9,13 @@ services:
 env:
   global:
     - LATEST="2.12-2.3.0"
+    - DOCKER_COMPOSE_VERSION=1.23.2
```

Comment:
Should probably indicate required version in the README / where to find this info. We should probably update info in the Dependencies section of the [ReleaseProcess](https://github.com/wurstmeister/kafka-docker/wiki/ReleaseProcess) wiki as well. 

Also, to me, it feels a little weird to pin docker-compose and not docker version. It seems that we either say this works with a specific environment and be explicit or we don't pin anything and work with "latest" (as defined by travis ci)